### DISCUSSION TITLE
Neustart des WR mit Akku zwischen Modulen und WR

### DISCUSSION THREAD

**Discussion by Gururom** [hoylabs/OpenDTU-OnBattery#1458] [Ideas]
Hallo,
sicher haben bereits mehrere Leute das gleiche Problem.

Habe seit kurzen zwei dieser Akkus, welche zwischen Modulen und Wechselrichter sitzen und diesem vorgaukeln sie wären Module.
Dadurch ist ein Weiterbetrieb der WR auch in sonnenloser Zeit möglich. (allinone-Lösung Marstek, Growatt, Bluepalm etc).
In diesem Fall bleibt in openDTUoB /DPL ja weiterhin der Punkt "Wechselrichter wird von Solarmodulen gespeist" aktiv.
Darum ist dann nicht möglich eine Neustartzeit für den WR festzulegen.
Kann man diese Möglichkeit nicht auch trotzdem einbinden?

beste Grüße
Stephan K.

**Comment by schlimmchen** [comment]
Was spricht dagegen, die WR im DPL als batteriegespeist einzustellen?

**Reply by spcqike** [reply]
Es gab bereits Probleme damit. Da die Batterien nicht die eigentliche Batterie Spannung ausgibt, sondern solar Module simuliert. Die Spannung kann man höher und niedriger sein. So zumindest hab ich es noch im Hinterkopf. 

Damals war die Lösung, die Batterien nicht als Batterie zu betrachten. Sondern eben als PV Modul. Was sie nach außen darstellen wollen. 

Welche Einstellungen und Spannungen würdest du als threshold empfehlen, damit man es nochmal als Batterie testet?

**Reply by Gururom** [reply]
Ja gibt's da Werte mit denen man das sicher testen kann.
Im Moment geht zwar sowieso nichts, wegen "dunkelflaute" da kann man nicht mal testen
